### DISCUSSION TITLE
What is our next step?

### DISCUSSION THREAD

**Discussion by fuksman** [anyproto/contributors#24] [Q&A]
Hey! 

First of all, I want to say how exciting it is to see the Contributors Community grow and your interest and desire to build Anytype together. 

We've already published our high-level plans about big projects for Open Any (https://github.com/orgs/anyproto/discussions/9), and the closest are: 
-  To start accepting contributions 
-  To launch a localization project 

But it's also crucial for us to understand your needs. For example:
- Should we prepare more documentation to simplify your onboarding as developers? What is the most important module to cover? 
- Should we elaborate more about the general architecture? 
- Should we host a Q&A/AMA session with tech leaders of Any? What project should be first? 

Please share your thoughts about what we should address and what is not yet covered in the Open Any Roadmap.

**Comment by hit-lacus** [comment]
I don't know if I have enough experience to share my opinion.

Writing tech blogs is a big project, and may cost you too much time. Maybe you can create a task/issue management system for different level contributors, publish some tasks and watch the feedback/reaction from the community.

Here is my simple draft, prepare and publish such a table/page won't cost you too much time.

| Level | Skill/Requirements | Suited task type | Guide  Link| Task Link |
|-------| ------------------ | -----------------| -----------|-----------|
| Newbie | Git & Github      | Translation      | A link     |A link where I cann find some uncompleted and suited issues |
| Junior | Some programming language like Go | Bug fix |   A link | A link where I cann find some uncompleted and suited issues |

If there are some task for newbie(but I never use Go language before), maybe I will have a try.
